tests/test_login.py

```diff
@@ -68,3 +68,35 @@ def test_get_session_user(self, client):
         retrieved_user_id = client.get_session_user(session_id=session_id)
 
         assert retrieved_user_id == user_id
+
+    def test_login_wrong_user(self, client):
+        try:
```

Comment:
This does not conform to the standard we use for checking exceptions in the hydra-base tests. 
We use the following format:
```python
with pytest.raises(hydra_base.exceptions.HydraLoginUserNotFound):
    user_id, session_id = client.login('wrong-user!', '')
```

Reply:
Please look at the other tests for reference

Reply:
Done it